product_secondary_unit/views/product_views.xml

```diff
@@ -24,4 +32,34 @@
             </xpath>
         </field>
     </record>
+
+    <record id="product_variant_easy_edit_view" model="ir.ui.view">
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_variant_easy_edit_view" />
+        <field name="groups_id" eval="[(4, ref('uom.group_uom'))]" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='weight']/.." position="after">
+                <field name="product_tmpl_id" invisible="1" />
+                <group string="Secondary Unit of Measure" name="secondary_uom">
```

Comment:
Should we remark this?
```suggestion
                <group string="Variant Spefecific Secondary Unit of Measure" name="secondary_uom">
```

Reply:
This view is already a variant so I think that write "Variant Spefecific... " is redundant..